Makefile

```diff
@@ -47,4 +52,13 @@ test-e2e: GO_TEST_FLAGS += -v
 test-e2e: GO_TEST_FLAGS += -timeout 2h
 test-e2e: GO_TEST_FLAGS += -p 1
 test-e2e: test-unit
+test-e2e:
+	go test \
```

Comment:
Thanks @geliu2016 - dunno if you saw this already, but @Elbehery added a very similar test in test/e2e/backup_script_test.go already. I believe you could reuse almost everything in there to write a testcase for that bug.

Reply:
Thank you, @tjungblu Yes, I saw there is similar test in backup_script_test.go already, and it checking the etcd backup feature with go function mode, and QE's test are focus on more complex scenarios, and QE using ginkgo mod to design test case, as I reviewed other sub component teams' e2e, they are also organize e2e test with go functions(Dev's prefer) and ginkgo(QE's prefer), it could co-exist, and I wish this backup case is a beginning, we will contribute QE's scenarios with this ginkgo framework.

Reply:
I prefer to have ginkgo tests in origin: https://github.com/openshift/origin/tree/master/test/extended/etcd

Otherwise this causes too much confusion and I don't want to maintain two test frameworks in here.